exercise_01.py

```diff
@@ -0,0 +1,19 @@
+def display_pattern(n):
```

Comment:
First, `display_pattern` that is a name that is too general. We know that this is a pyramid of numbers. Second, you are not displaying anything nor printing, so why are you lying to the user. What you are doing is creating a string which is a pyramid of numbers, so think about an appropriate number for this function.